### DISCUSSION TITLE
guide/drivers/115

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#80] [Announcements]
# guide/drivers/115

Due to the limitation of 115 Cloud, the download must carry cookies, so only the proxy function in this program can be used for transfer. (You can use other machines to transfer...

https://alist.nn.ci/guide/drivers/115.html

**Comment by Lzp119520** [comment]
The `cookie` is obtained from the request.

**Comment by jiecaosanluomand** [comment]
g f f g g
